### ISSUE TITLE
Remove Harasser

### ISSUE THREAD

**Issue by ShadowGoat** [ContributorCovenant/contributor_covenant#595]
CoralineAda is harassing members of other projects. She does not value others as human beings and actively hampers an atmosphere of kindness, cooperation, and understanding if she does not agree with there personal offsite activity.

For the reasons of personal attacks, trolling, public harassment, insults, and other unprofessional conduct I request she be removed.

Copied from another thing coralline closed.

**Comment by falkTX** [comment]
> Is @CoralineAda really right person to carry on this project?

Yes, dealing with your constant bullshit requires attitudes like those.
As I see it, you are getting exactly what you deserve.

**Comment by liltinkerer** [comment]
My constant bullshit? What are you talking about and why are you attacking me?

**Comment by falkTX** [comment]
I mean you in the plural form, not you personally, sorry.
